app/config/index.js

```diff
@@ -8,12 +10,26 @@ module.exports = {
     secret: process.env.SECRET,
   },
   database: {
-    name: process.env.DB_NAME,
-    username: process.env.DB_USER,
-    password: process.env.DB_PASSWORD,
-    host: process.env.DB_HOST,
-    dialect: process.env.DB_DIALECT || 'postgres',
-    port: process.env.DB_PORT || 5432,
+    development: {
+      name: process.env.DB_NAME,
+      username: process.env.DB_USER,
+      password: process.env.DB_PASSWORD,
+      host: process.env.DB_HOST,
+      dialect: process.env.DB_DIALECT || 'postgres',
+      port: process.env.DB_PORT || 5432,
+    },
+    staging: {
+      name: process.env.STAGING_DB_NAME,
+      username: process.env.STAGING_DB_USER,
+      password: process.env.STAGING_DB_PASSWORD,
+      host: process.env.STAGING_DB_HOST,
+      dialect: process.env.STAGING_DB_DIALECT || 'postgres',
+      port: process.env.STAGING_DB_PORT || 5432,
+      dialectOptions: {
+        ssl: true,
+        ca: readFileSync(resolve(__dirname, '../keys', 'certificate.crt')),
```

Comment:
Not strictly required, but is nice to have I suppose. Just have to document where the key should be. Also, the SSL should toggle based on `server.environment === "staging"` (or something)